src/main/java/com/xellitix/jenkins/plugintool/authentication/JenkinsApiTokenRetriever.java

```diff
@@ -0,0 +1,19 @@
+package com.xellitix.jenkins.plugintool.authentication;
+
+import com.xellitix.jenkins.plugintool.cli.command.FetchInstalledPluginsCommand;
+
+/**
+ * Jenkins API token retriever interface.
+ *
+ * @author Garrett Ewens
+ */
+public interface JenkinsApiTokenRetriever {
+
+  /**
+   * Gets the API token.
+   *
+   * @param command The {@link FetchInstalledPluginsCommand}.
+   * @return The API token.
+   */
+  String getApiToken(FetchInstalledPluginsCommand command);
```

Comment:
* Add "throws JenkinsApiTokenNotFoundException" to your method signature.
* Document the exception in the javadoc